.github/workflows/ci-tests.yaml

```diff
@@ -45,10 +40,25 @@ jobs:
           name: "py${{ matrix.python-version }}"
           token: ${{ secrets.CODECOV_TOKEN }}
 
+  test-pandas:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.12"
+      - name: Install dependencies
+        run: |
+          pip install -e '.[dev]'
```

Comment:
This job is essentially the same as the one above, but we only install `dev` dependencies, rather than `all`